src/pybind/mgr/dashboard/services/auth/auth.py

```diff
@@ -62,8 +62,9 @@ def from_db(cls, db: Optional['SsoDB'] = None) -> Type["BaseAuth"]:
     class Config(TypedDict):  # pylint: disable=inherit-non-class
         pass
 
+    @staticmethod
     @abc.abstractmethod
-    def to_dict(self) -> 'Config':
+    def to_dict() -> 'Config':
         pass
```

Comment:
Nono, you didn't understand me. As I commented there, the `to_dict` obviously has to be an instancemethod, but it cannot return something from another class as it did and still does for the `Local` subclass.

Reply:
Ah okay, fixed it